kpatch-build/lookup.c

```diff
@@ -262,6 +283,8 @@ struct lookup_table *lookup_open(char *obj_path, char *symvers_path,
 		ERROR("malloc table");
 	memset(table, 0, sizeof(*table));
 
+	table->vmlinux = !strcmp(basename(obj_path), "vmlinux");
+
```

Comment:
I don't have an opinion either way, though I would at least argue that it shouldn't be in the `if (locals)` condition block since it might end up being useful in other places someday.

BTW, looking at the code, the `table->local_syms = NULL` and `if (locals)` check should probably be moved to inside `find_local_syms()` because they're low-level details involved in finding the locals.  I'll tack on a commit to this PR to do that.